rudder-core/src/main/scala/com/normation/rudder/reports/ReportingConfiguration.scala

```diff
@@ -37,12 +37,22 @@ package com.normation.rudder.reports
 import com.normation.utils.HashcodeCaching
 import net.liftweb.json.JBool
 import net.liftweb.common._
+import org.joda.time.Duration
+import com.normation.inventory.domain.NodeId
+import com.normation.inventory.domain.NodeId
+import com.normation.rudder.services.nodes.NodeInfoService
+import com.normation.rudder.domain.Constants
 
 /**
  * Class that contains all relevant information about the reporting configuration:
  * * agent run interval
  * * compliance mode (soon)
  * * heartbeat configuration
+ *
+ * Everything is option because that information is at the node level and
+ * can not be defined (for example if the node only use default global values)
```

Comment:
Could you change "can" for "may"; at first i thought it was illegal to define it